src/components/sidebar/helpers/generate-resources-nav-items.ts

```diff
@@ -19,6 +19,34 @@ const generateCommunityForumNavItem = (productSlug?: ProductSlug) => {
   return { href, title }
 }
 
+/**
+ * A dictionary of product slugs to GitHub links.
+ */
+const GITHUB_LINKS_BY_SLUG: { [K in ProductSlug]: string } = {
+  boundary: 'https://github.com/hashicorp/boundary',
+  consul: 'https://github.com/hashicorp/consul',
+  hcp: 'https://github.com/hashicorp', // No specific repo to link to
+  nomad: 'https://github.com/hashicorp/nomad',
+  packer: 'https://github.com/hashicorp/packer',
+  sentinel: 'https://github.com/hashicorp', // No specific repo to link to
+  terraform: 'https://github.com/hashicorp/terraform',
+  vagrant: 'https://github.com/hashicorp/vagrant',
+  vault: 'https://github.com/hashicorp/vault',
+  waypoint: 'https://github.com/hashicorp/waypoint',
```

Comment:
💭 Since a default is being provided that matches what's specified for `hcp` and `sentinel`, I think we can remove those from this map.

```suggestion
  boundary: 'https://github.com/hashicorp/boundary',
  consul: 'https://github.com/hashicorp/consul',
  nomad: 'https://github.com/hashicorp/nomad',
  packer: 'https://github.com/hashicorp/packer',
  terraform: 'https://github.com/hashicorp/terraform',
  vagrant: 'https://github.com/hashicorp/vagrant',
  vault: 'https://github.com/hashicorp/vault',
  waypoint: 'https://github.com/hashicorp/waypoint',
```

Reply:
We could, but then I think we'd need to adjust the type here! Would you be okay with keeping it as is?